File: fstp-core-bo/src/main/java/com/purefun/fstp/core/bo/CommandBO.java
package com.purefun.fstp.core.bo;

import com.purefun.fstp.core.bo.tool.fstpbo;

@fstpbo(boid = 999L, destination = "fstp.core.manager.command")
public class CommandBO extends BaseBO{
	public String targetService = "";
	public String parm0 = "";
	public String parm1 = "";
	public String parm2 = "";
	public String parm3 = "";
	public String parm4 = "";	
}

File: fstp-python-2.7/src/com/purefun/fstp/core/bo/model/CommandBO.py
import uuid

class CommandBO(object):

    def __init__(self):
        self.targetService = ''
        self.parm0 = ''
        self.parm1 = ''
        self.parm2 = ''
        self.parm3 = ''
        self.parm4 = ''
        self.uuid = str(uuid.uuid1())
        self.boid = 999
        self.destination = "fstp.core.manager.command"
